docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -63,216 +37,137 @@ $ # Log out and log back in again to pick up the group changes
 $ exit
 ```
 
-#### Download and unpack Alluxio
+## Launch Alluxio
 
-1. [Download](https://www.alluxio.org/download) Alluxio and copy it to the linux machine.
-1. Unpack the Alluxio tarball to a directory.
 ```bash
-$ tar xvfz alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-<hadoop distribution>.tar.gz
+# Launch the Alluxio master
+$ docker run -d --net=host
+    -v /mnt/data:/opt/alluxio/underFSStorage
```

Comment:
comment that /mnt/data must exist on the host machine and that we will use that as root ufs for this tutorial.

Reply:
moved the below comment to before the commands